mne/annotations.py

```diff
@@ -453,6 +453,14 @@ def save(self, fname, *, overwrite=False, verbose=None):
             The filename to use.
         %(overwrite)s
 
+        Notes
+        -----
+        The format of the information stored in the saved annotation objects
+        depends on the chosen file format. :file:`.csv` files store the onset
+        as timestamps (e.g. `2002-12-03 19:01:56.676071`), whereas :file:`.txt`
+        files store onset as seconds since start of the recording (e.g.
+        `45.95597082905339`).
```

Comment:
suggestion: use code formatting instead of italics, and apply comma after e.g.

```suggestion
        as timestamps (e.g., ``2002-12-03 19:01:56.676071``),
        whereas :file:`.txt` files store onset as seconds since start of the
        recording (e.g., ``45.95597082905339``).
```